.travis.yml

```diff
@@ -3,27 +3,20 @@ dist: focal
 language: python
 python: 3.8
 cache: pip
-before_install: pip install --upgrade pip setuptools six
-install: pip install black flake8
+install:
+  - pip install --upgrade pip setuptools six
+  - pip install -r requirements_pytest.txt
 jobs:
   include:
     - name: Build
-      before_script:
-        - black --check . || true
-        - flake8 --ignore=E203,W503 --max-complexity=25 --max-line-length=88 --statistics --count .
-        - scripts/validate_filenames.py  # no uppercase, no spaces, in a directory
-        - pip install -r requirements.txt  # fast fail on black, flake8, validate_filenames
       script:
-        - mypy --ignore-missing-imports . || true  # https://github.com/python/mypy/issues/7907
         - pytest --doctest-modules --ignore=project_euler/ --durations=10 --cov-report=term-missing:skip-covered --cov=. .
     - name: Project Euler
-      before_script:
-        - pip install pytest-cov pytest-subtests
-        - pytest --tb=no --no-summary --capture=no project_euler/validate_solutions.py || true  # fail fast on wrong solution
       script:
+        - pytest --tb=no --no-summary --capture=no project_euler/validate_solutions.py || true  # fail fast on wrong solution
         - pytest --doctest-modules --durations=10 --cov-report=term-missing:skip-covered --cov=project_euler/ project_euler/
```

Comment:
By keeping the testing of Project Euler solutions in `script` section, Travis won't exit if it fails. If the test is in `before_script` section, Travis will exit.

Reply:
Why do we run that `project_euler/validate_solutions.py` thru pytest?  It seems like a lot of complexity and overhead.

Reply:
I did do it without `pytest` but you mentioned (https://github.com/TheAlgorithms/Python/pull/2471#issuecomment-698533067) to include it in as that is already included.

Currently, the algorithm is set in such a way to utilize the `pytest.mark.parameterize` decorator which calls the function with the arguments as `problem_number` and `expected` and at the end uses `pytest.fixture` to print all the error messages. This can be done without using `pytest`.

Reply:
It would be similar to this without `pytest`: https://github.com/TheAlgorithms/Python/blob/f1d60aca9e8c6cba013e5e5571d180f062bb3452/project_euler/validate_solutions.py

The script will be self-contained and will need no dependencies.

Reply:
It was run automatically by pytest when its name was `project_euler/solution_test.py` but that stopped being the case when the filename was changed to something that does not obey pytest discovery rules.  Let's not mess with it for now because we ___need___ to run in `|| true` mode.

Reply:
Are you OK with the current state of this PR or is there something that I should change?

Reply:
I think `validate_solutions.py` should be in `before_script` section so that it fails fast when we remove `|| true`